var/spack/repos/builtin/packages/xterm/package.py

```diff
@@ -12,9 +12,13 @@ class Xterm(AutotoolsPackage):
     that can't use the window system directly."""
 
     homepage = "http://invisible-island.net/xterm/"
-    url      = "ftp://ftp.invisible-island.net/xterm/xterm-327.tgz"
+    url      = "https://github.com/ThomasDickey/xterm-snapshots/archive/xterm-353.tar.gz"
```

Comment:
The current `url` works fine. Why changing it?

Reply:
It's just the same as #14835 , maybe some problem with the network in my side.
I will close this PR.

Reply:
It still need some version update, I will restore the url and checksum for serveral new versions.